### PR TITLE
Ftr: return the total length and write length

### PR DESCRIPTION
**Pull request by cvictory**
return the total length and write length in WritePkg

### CODE REVIEW

**Review by watermelo — APPROVED** [review]

File: client_test.go [hunk 1]
```diff
@@ -170,8 +170,10 @@ func TestTCPClient(t *testing.T) {
 
 func TestUDPClient(t *testing.T) {
 	var (
-		err  error
-		conn *net.UDPConn
+		err      error
+		conn     *net.UDPConn
+		leng     int
```
Comment (AlexStocks) [inline client_test.go:175]: leng -> sendLength

File: getty.go [hunk 2]
```diff
@@ -171,7 +171,9 @@ type Session interface {
 
 	// the Writer will invoke this function. Pls attention that if timeout is less than 0, WritePkg will send @pkg asap.
 	// for udp session, the first parameter should be UDPContext.
-	WritePkg(pkg interface{}, timeout time.Duration) error
+	// total: need send total bytes
```
Comment (AlexStocks) [inline getty.go:174]: // WritePkg: encode package @pkg to bytes steam and send them out
//
// totalBytesLength: @pkg stream bytes length after encoding @pkg.
// sendBytesLength: stream bytes length that sent out successfully.
// error: send out error from system

File: client_test.go (hunk 1 above)
Comment (cvictory) [reply client_test.go]: done

File: getty.go (hunk 2 above)
Comment (cvictory) [reply getty.go]: done

**Review by AlexStocks — APPROVED** [review]

**Review by cityiron — APPROVED** [review]

**Pull request merged**